src/components/themed/WalletListCurrencyRow.tsx

```diff
@@ -50,13 +55,40 @@ const WalletListCurrencyRowComponent = (props: Props) => {
 
   return (
     <TouchableOpacity accessible={false} style={styles.row} onLongPress={handleLongPress} onPress={handlePress}>
-      {customAsset != null ? <CustomAssetRow customAsset={customAsset} /> : <CurrencyRow showRate={showRate} token={token} tokenId={tokenId} wallet={wallet} />}
+      {isPaused ? (
+        <>
+          <View style={styles.overlayLabelContainer}>
+            <EdgeText style={styles.overlayLabel}>{lstrings.fragment_wallets_wallet_paused}</EdgeText>
+          </View>
+          <View style={styles.overlayContainer} />
+        </>
+      ) : null}
+      {customAsset != null ? (
+        <CustomAssetRow customAsset={customAsset} />
+      ) : (
+        <CurrencyRow showRate={showRate && !isPaused} token={token} tokenId={tokenId} wallet={wallet} />
+      )}
     </TouchableOpacity>
   )
 }
 
 const getStyles = cacheStyles((theme: Theme) => ({
-  // Layout:
+  overlayLabel: {
+    color: theme.overlayDisabledTextColor
+  },
+  overlayLabelContainer: {
+    ...StyleSheet.absoluteFillObject,
+    alignItems: 'center',
+    display: 'flex',
+    justifyContent: 'center',
+    zIndex: 2
+  },
+  overlayContainer: {
+    ...StyleSheet.absoluteFillObject,
+    backgroundColor: theme.overlayDisabledColor,
+    opacity: theme.overlayDisabledOpacity,
+    zIndex: 1
+  },
```

Comment:
Just put the background color on `overlayLabelContainer`, but use either `RGBA(0, 0, 0, 0.5)` or `#00000080` to put the opacity into the color itself, removing the need for `theme.overlayDisabledOpacity` as well.

Reply:
Just realized we have that in the palette